mobify/platform-scaffold
# Remove !important to utilities

Remove `!important` styles to utilities. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-38
 **Linked PRs**: N/A

## Changes
- change `fadeIn` to `fade-in`
- change `inlineLoaderPulseAnimation` to `inline-loader-pulse`
- move animation styles in its own files
- new mixin: `wrap-at-root`
- apply `wrap-at-root` to utilities
- remove `!important` styles to utilities 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Check all pages to make sure that there are no broken styles
